model/messages.go

```diff
@@ -0,0 +1,123 @@
+package model
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/pkg/errors"
+)
+
+type BanTx struct {
+	UserName string `json:"username"`
+}
+
+// Message represents a message sent by a user
+type Message struct {
+	Sender  string `json:"sender"`
+	Message string `json:"message"`
+}
+
+type MsgHistory struct {
+	Msg string `json:"history"`
```

Comment:
Is this the history of all messages? If so, we could have a first version of an appHash by hashing this message (I believe there is only one in the DB)

Reply:
Yes, that is the more correct way to do it. I followed what was done in the kvstore , but this would be more correct.